tests/migration/migration.go

```diff
@@ -293,7 +293,7 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				// check VMI, confirm migration state
 				libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
 			})
-			It("should be successfully migrate with a WriteBack disk cache", func() {
+			It("should be successfully migrate with a WriteBack disk cache", decorators.Conformance, func() {
```

Comment:
Imho the disk cache should not have major implications on the migration and so I would not make it conformance but let's re-evaluate in follow up

Reply:
It was removed for the moment.